redaxo/src/core/lib/list.php

```diff
@@ -345,6 +355,60 @@ public function getLinkAttributes($column, $default = null)
         return $this->linkAttributes[$column] ?? $default;
     }
 
+    // row attribute setter/getter etc
+
+    /**
+     * Methode, um dem der Zeile (<tr>) Attribute hinzuzufügen.
+     *
+     * @param mixed $attr   entweder ein array: [attributname => attribut, ...]
+     *                      oder eine Callback-Funktion
+     */
+    public function setRowAttributes($attr)
+    {
+        if (is_callable($attr)) {
+            $this->rowAttributes = $attr;
+            $this->rowAttributesCallable = true;
+        } elseif (is_array($attr) && count($attr)) {
+            $this->rowAttributes = $attr;
+            $this->rowAttributesCallable = false;
+        } else {
+            $this->rowAttributes = [];
+            $this->rowAttributesCallable = false;
+        }
+    }
+
+    /**
+     * Methode, um die aktuellen Attribute der Zeile (<tr>) Attribute abzufragen.
+     *
+     * @return mixed    entweder ein array: [attributname => attribut, ...]
+     *                  oder eine Callback-Funktion
+     */
+    public function getRowAttributes()
```

Comment:
wozu brauchen wir den getter? in den beispielen und dem code hier scheint er nicht notwendig zu sein?

Reply:
Ersten gibt es für Spaltenformate auch Getter, also ist es konsistent. Zweitens ist es auch sinnvoll, z.B. um erst abzufragen, was evtl. schon an andere Stelle gesetzt wurde und dann weitere Einstellungen hinzuzufügen. Kommt jetzt nicht soo oft vor in freier Wildbahn, aber bei Columns hab ich so eine Kaskadierung im YForm-Context schon mal genutzt.